ui/mirage/scenarios/default.js

```diff
@@ -44,14 +46,74 @@ function smallCluster(server) {
   server.create('feature', { name: 'Dynamic Application Sizing' });
   server.createList('agent', 3, 'withConsulLink', 'withVaultLink');
   server.createList('node', 5);
-  server.createList('job', 5, { createRecommendations: true });
+  server.createList('job', 1, { createRecommendations: true });
   server.createList('allocFile', 5);
   server.create('allocFile', 'dir', { depth: 2 });
   server.createList('csi-plugin', 2);
 
+  // #region evaluations
+
+  // Branching: a single eval that relates to N-1 mutually-unrelated evals
+  const NUM_BRANCHING_EVALUATIONS = 3;
```

Comment:
`nit` `question` `non-blocking` should this move to the top of the file or a `constants` file in the `test/utils` folder?